examples/raw.rs

```diff
@@ -6,7 +6,7 @@
 mod common;
 
 use crate::common::parse_args;
-use tikv_client::{raw::Client, Config, Key, KvPair, Result, ToOwnedRange, Value};
+use tikv_client::{Config, Key, KvPair, RawClient as Client, Result, ToOwnedRange, Value};
```

Comment:
I gotta say, I really don't think this is better than `raw::Client`. :(

Reply:
Yeah, it might be reasonable to change it back. I felt it was a bit odd making people use a module for just a couple of things, but it is also sub-optimal to have RawClient and TransactoinClient when you should only use one.

Reply:
When you should **definitely** only use one.